src/include/daos_srv/bio.h

```diff
@@ -849,11 +849,13 @@ void *bio_buf_addr(struct bio_desc *biod);
  * \param umem		[IN]	umem instance
  * \param bsgl_src	[IN]	Source BIO SGL
  * \param bsgl_dst	[IN]	Target BIO SGL
+ * \param desc		[OUT]	Returned BIO copy descriptor
```

Comment:
Just a note, this syntax is `like` doxygen code annotations but isn't doxygen, I thought I'd updated all of these to be actual doyxgen.